Readme.md

```diff
@@ -1,19 +1,30 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128593320/23.1.6%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T473819)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* **[CustomDashboardDesignerFormController.cs](CS/EF/DashboardCustomFormEF/DashboardCustomFormEF.Win/Controllers/CustomDashboardDesignerFormController.cs)**
-* [CustomDashboardDesignerForm.cs](CS/EF/DashboardCustomFormEF/DashboardCustomFormEF.Win/CustomDashboardDesignerForm.cs)
-<!-- default file list end -->
-# How to show a custom form as the WinForms Dashboard Designer
+# XAF - How to show a custom form as the WinForms Dashboard Designer
```

Comment:
I have a question about the definite article before WinForms Dashboard Designer. Shouldn't it be "a"?

Reply:
```suggestion
# XAF - How to show a custom form as a WinForms Dashboard Designer
```